vocab_list/tests.py

```diff
@@ -17,8 +17,17 @@
 bin\tbash
 hello\tworld"""
 
+dupe_vocab_list_tsv_content = b"""\
+car\tarmor
+car\tarmor"""
+
+dupe_headword_list_tsv_content = b"""\
+nintendo\tmario
+nintendo\tmario2"""
```

Comment:
It might be instructive to use `quam` for this test since it's a real example of a word with different definitions:

- **quam** as possible as
- **quam** how
- **quam** than